webutil/etag.go

```
@@ -3,16 +3,11 @@ package webutil
 import (
 	"crypto/md5"
 	"encoding/hex"
-
-	"github.com/blend/go-sdk/ex"
 )
 
 // ETag creates an etag for a given blob.
-func ETag(contents []byte) (string, error) {
+func ETag(contents []byte) string {
 	hash := md5.New()
-	_, err := hash.Write(contents)
-	if err != nil {
-		return "", ex.New(err)
-	}
-	return hex.EncodeToString(hash.Sum(nil)), nil
+	hash.Write(contents)
```

ISTM `_, _ = hash.Write(contents)` is the better move here, with a comment explaining why it's safe to ignore the error.

If we ever decide to integrate [`errcheck`][1] into CI (which I highly recommend) this will get flagged.

[1]: https://github.com/kisielk/errcheck